migration_spinner/spinner.py

```diff
@@ -0,0 +1,40 @@
+import os
+import time
+
+from airflow import settings, version
```

Comment:
Though if we're going to import airflow anyway then we can just use `airflow.__file__`